src/data/thermal-conductivity-constants.ts

```diff
@@ -411,6 +411,726 @@ export const DIPPRThermalConductivityDictionary: Record<
+    thermalConductivityAtMinimumTemperature: 0.1815,
+    thermalConductivityAtMaximumTemperature: 0.1412,
+  },
+
+  CyclohexylMercaptan: {
+    name: 'Cyclohexyl mercaptan',
+    formula: 'C6H12S',
+    CAS: '1569-69-3',
+    molecularWeight: 116.22448,
+    C1: 0.18374,
+    C2: -0.0001925,
+    minimumTemperature: 189.64,
+    maximumTemperature: 431.95,
+    thermalConductivityAtMinimumTemperature: 0.1472,
+    thermalConductivityAtMaximumTemperature: 0.1006,
+  },
+
+  Decanal: {
+    name: 'Decanal',
+    formula: 'C10H20O',
+    CAS: '112-31-2',
+    molecularWeight: 156.2652,
+    C1: 0.21363,
+    C2: -0.00023004,
+    minimumTemperature: 285.0,
+    maximumTemperature: 481.65,
+    thermalConductivityAtMinimumTemperature: 0.1481,
+    thermalConductivityAtMaximumTemperature: 0.1028,
+  },
+
+  Decane: {
+    name: 'Decane',
+    formula: 'C10H22',
+    CAS: '124-18-5',
+    molecularWeight: 142.28168,
+    C1: 0.2063,
+    C2: -0.00025,
+    minimumTemperature: 243.51,
+    maximumTemperature: 447.3,
+    thermalConductivityAtMinimumTemperature: 0.1454,
+    thermalConductivityAtMaximumTemperature: 0.0945,
+  },
+
+  DecanoicAcid: {
+    name: 'Decanoic acid',
+    formula: 'C10H20O2',
+    CAS: '334-48-5',
+    molecularWeight: 172.265,
+    C1: 0.206,
+    C2: -0.0002,
+    minimumTemperature: 304.75,
+    maximumTemperature: 543.15,
+    thermalConductivityAtMinimumTemperature: 0.1451,
+    thermalConductivityAtMaximumTemperature: 0.0974,
+  },
+
+  '1Decanol': {
+    name: '1-Decanol',
+    formula: 'C10H22O',
+    CAS: '112-30-1',
+    molecularWeight: 158.28108,
+    C1: 0.236171,
+    C2: -0.00025,
+    minimumTemperature: 280.05,
+    maximumTemperature: 503.0,
+    thermalConductivityAtMinimumTemperature: 0.1662,
+    thermalConductivityAtMaximumTemperature: 0.1104,
+  },
+
+  '1Decene': {
+    name: '1-Decene',
+    formula: 'C10H20',
+    CAS: '872-05-9',
+    molecularWeight: 140.2658,
+    C1: 0.20237,
+    C2: -0.00024187,
+    minimumTemperature: 206.89,
+    maximumTemperature: 443.75,
+    thermalConductivityAtMinimumTemperature: 0.1523,
+    thermalConductivityAtMaximumTemperature: 0.095,
+  },
+
+  DecylMercaptan: {
+    name: 'Decyl mercaptan',
+    formula: 'C10H22S',
+    CAS: '143-10-2',
+    molecularWeight: 174.34668,
+    C1: 0.20134,
+    C2: -0.00020826,
+    minimumTemperature: 247.56,
+    maximumTemperature: 512.35,
+    thermalConductivityAtMinimumTemperature: 0.1498,
+    thermalConductivityAtMaximumTemperature: 0.0946,
+  },
+
+  '1Decyne': {
+    name: '1-Decyne',
+    formula: 'C10H18',
+    CAS: '764-93-2',
+    molecularWeight: 138.24992,
+    C1: 0.20839,
+    C2: -0.00023622,
+    minimumTemperature: 229.15,
+    maximumTemperature: 447.15,
+    thermalConductivityAtMinimumTemperature: 0.1543,
+    thermalConductivityAtMaximumTemperature: 0.1028,
+  },
+
+  Deuterium: {
+    name: 'Deuterium',
+    formula: 'D2',
+    CAS: '7782-39-0',
+    molecularWeight: 4.0316,
+    C1: 1.264,
+    C2: NaN,
```

Comment:
```suggestion
```

Update the interface and formula if needed.

Reply:
Sir ,C2 column was missing only  for this compound . so I think it is good it if we keep it .

Reply:
The interface should closely define the nature of the values. Not updating the interface can be misleading because the reader may feel that `C2` will **always** be defined, which is not the case.

Moreover, `NaN` is not a good value keeping in mind the concerned formula in `calculateDIPPRThermalConductivity` function.

Reply:
ok I will update the interface and remove this value